pulpcore/app/serializers/repository.py

```diff
@@ -407,25 +407,21 @@ class RepositoryVersionSerializer(ModelSerializer, NestedHyperlinkedModelSeriali
         view_name='versions-detail',
         lookup_field='number', parent_lookup_kwargs={'repository_pk': 'repository__pk'},
     )
-    _content_href = NestedIdentityField(
-        view_name='versions-content',
-        lookup_field='number', parent_lookup_kwargs={'repository_pk': 'repository__pk'},
+    content = serializers.SerializerMethodField(
```

Comment:
As I mentioned on IRC, we should be consider leaving some space for future expansion should it be necessary. 

I suppose if we ever needed to add back in the content urls, we could use "_hrefs" as we did previously.  But we could consider other alternatives.